accounts/serializers.py

```
@@ -134,3 +134,23 @@ class Meta:
             "created_at",
             "updated_at",
         ]
+
+
+class SimpleProfileDetailSerializer(serializers.ModelSerializer):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)